velox/dwio/parquet/reader/ParquetData.cpp

```diff
@@ -85,8 +85,10 @@ void ParquetData::enqueueRowGroup(
   }
   VELOX_CHECK_GE(chunkReadOffset, 0);
 
-  uint64_t readSize = std::min(
-      metaData.total_compressed_size, metaData.total_uncompressed_size);
+  uint64_t readSize = (metaData.total_uncompressed_size > 0)
```

Comment:
DuckDB does not set `total_uncompressed_size`. Hence this is required.

Reply:
`metaData.__isset.total_uncompressed_size`?